libs/barista-components/select/src/select.ts

```diff
@@ -108,7 +109,7 @@ let uniqueId = 0;
 const LOG: DtLogger = DtLoggerFactory.create('DtSelect');
 
 /** The height of the select items. */
-export const SELECT_ITEM_HEIGHT = 32;
+export const SELECT_ITEM_HEIGHT = 28;
```

Comment:
may we use a css custom property with that that it is in sync with the css property?

Reply:
I'd postpone this for now - but i like this idea a lot!